<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNaverSearch</class>
 <widget class="QWidget" name="frmNaverSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕코딩</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>네이버 영화 검색</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>영화 제목 검색</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>75</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>검색어</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSearch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>28</y>
      <width>93</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>검색</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tblResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>86</y>
     <width>459</width>
     <height>386</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
